content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,37 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Hardware recommendations and considerations {#windows-hardware-recommendations}
+
+{{< note >}}
+The following hardware requirements are _recommended_ values and do not represent
+minimum requirements. Refer to [Production worker nodes Kubernetes documentation](https://kubernetes.io/docs/setup/production-environment/#production-worker-nodes)
+for guidance on deciding on resources for production worker nodes.
+{{< /note >}}
+
+- 64-bit processor 2 CPU cores or more, capable of supporting virtualization
+- 8GB or more of RAM
+- 50GB or more of free disk space
+
+Refer to the specific Windows OS documentation for the most
+up-to-date information on minimum hardware requirements. For Windows Server
+installations, refer to
+[Hardware requirements for Windows Server Microsoft documentation](https://learn.microsoft.com/en-us/windows-server/get-started/hardware-requirements)
+for the bare minimum hardware requirements. 
+
+To optimize system resources, if a graphical user interface is not required,
+it may be preferable to use a Windows OS installation that excludes
```

Comment:
nit: Windows -> Windows Server, for sake of explicit and to avoid confusion w/ Windows 11